src/database/__tests__/Database.test.ts

```diff
@@ -238,13 +243,28 @@ describe("Database implementations", () => {
         const indexedOrder = forgeIndexedOrder();
         await db.addOrder(indexedOrder);
 
-        await db.deleteOrder("hash");
+        await db.deleteOrder("nonce", "0x0000000000000000000000000000000000000000");
```

Comment:
Make global constant for `"0x0000000000000000000000000000000000000000"` or `import { AddressZero } from '@ethersproject/constants';`

Reply:
Done thank you :)